handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/FallbackPolicyManager.java

```diff
@@ -39,6 +42,7 @@ public static void record(String type, Invocation invocation, Response response,
   public static Response getFallbackResponse(String type, Throwable error, Invocation invocation) {
     FallbackPolicy policy = getPolicy(type, invocation);
     if (policy != null) {
+      EventManager.post(new FallbackEvent(policy, invocation, Type.OPEN));
```

Comment:
Fallback is called in HystrixCommand, send event there. But I think this will send a lot of unnecessary events because any error will call fallback.